docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -289,8 +289,8 @@ documentation.
 **Note:** Headers are appended to requests/responses in the following order: route level headers, virtual host
 level headers and finally global level headers.
 
-Envoy additionally supports adding dynamic values to the **request** headers. Supported dynamic
-values are:
+Envoy additionally supports adding dynamic values to **request** and **response** headers.
```

Comment:
nit: I only bolded this in the first place because it was request only. I would probably unbold. cc @zuercher

Reply:
I'll try to remember fix this when I update this document for envoyproxy/envoy#2148.